pubspec.yaml

```diff
@@ -23,8 +23,8 @@ flutter:
         pluginClass: FlutterNsdPlugin
       ios:
         pluginClass: SwiftFlutterNsdPlugin
+      macos:
+        pluginClass: SwiftFlutterNsdPlugin
       windows:
         fileName: flutter_nsd_plugin.cpp
         pluginClass: FlutterNsdPlugin
-      macos:
-        pluginClass: SwiftFlutterNsdPlugin
```

Comment:
Thanks once again for being proactive :) Could you please create a new branch out of origin/master before creating a PR? This will make sure that old commits won't show up in the PR and prevent unrelated changes such as this one from appearing in the PR :)

What I suggest is that you close this PR, create the new branch from latest `origin/master` and then `cherry-pick` the last commits from the old `windows` branch.

Thank you 🙏

Reply:
OK ill give it a try. I am not a real git expert so bear with me...